packages/salesforcedx-vscode-core/src/orgBrowser/metadataCmp.ts

```diff
@@ -29,13 +31,21 @@ export class ComponentUtils {
       (await OrgAuthInfo.getUsername(defaultUsernameOrAlias)) ||
       defaultUsernameOrAlias;
 
+    let fileName = metadataType;
+    if (TypeUtils.FOLDER_TYPES.has(metadataType)) {
```

Comment:
Is it possible to just check if a folder is specified instead of also verifying the metadata is a folder type? Is there any scenario in which this method could be called with a folder specified without actually being a folder type? It seems like this could only be triggered with a folder once it passes the check in 'getComponents' of the outline provider.